bridges/ethElrond/steps/elrondToEth/step01GetPending.go

```diff
@@ -16,6 +16,7 @@ type getPendingStep struct {
 func (step *getPendingStep) Execute(ctx context.Context) core.StepIdentifier {
 	step.bridge.ResetRetriesCountOnEthereum()
 	step.bridge.ResetRetriesCountOnElrond()
+	step.bridge.ResetRetriesOnWasTransferProposedOnElrond()
```

Comment:
```suggestion
	step.bridge.ResetRetriesCountersOnElrond()
```
maybe have only on function to reset all counters on one chain